tests/Integrations/Custom/Autoloaded/FrameworkNoAutoloaderTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
Since I would expect autoloaded tests to be in the `Autoloaded` folder, could we possibly create a new folder called `tests/Integrations/Custom/NotAutoloaded` and move this test to it? :)